src/style.css

```diff
@@ -1 +1,69 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+ul {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px 50px;
+}
+
+.nav {
+  display: flex;
+  margin-right: 30px;
+}
+
+.nav__link {
+  font-family: inherit;
+  position: relative;
+  align-items: center;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+  height: 60px;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  width: 100%;
+  height: 4px;
+  background: #00acdc;
+  border-radius: 8px;
+  left: 0;
+  top: 34px;
+}
+
+.nav__item {
+  display: inline-block;
+  list-style-type: none;
+  margin-left: 20px;
+  padding-top: 19px;
+  padding-bottom: 19px;
```

Comment:
Paddings should be added to the links, not li, so links are clickable for the whole height